content/blog/2020/05/2020-05-31-machine-learning-plugin-community-bonding.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Machine Learning Plugin"
```

Comment:
Would it be possible to change the blog title to something more expliclt, e.g. 

```suggestion
title: "Machine Learning Plugin project - community bonding report"
```

Or something like that

Reply:
Yes of course @oleg-nenashev